import-tests/tests/test_ops.py

```diff
@@ -416,6 +415,8 @@ def test_mathtransform():
         # {"opName":"max_pooling1d", "outName":"max_pooling1d/channels_last_b2_k2_s1_VALID", "varShapes":[[2, 5, 2]], "varTypes":["float32"], "pooling_size":2, "stride":1, "padding":"VALID", "data_format":"channels_last"},
         # {"opName":"max_pooling1d", "outName":"max_pooling1d/channels_last_b1_k2_s2_SAME", "varShapes":[[1, 5, 2]], "varTypes":["float32"], "pooling_size":2, "stride":2, "padding":"SAME", "data_format":"channels_last"},
 
+        #{"opName":"max_pool_with_argmax", "outName":"max_pool_with_argmax/rank1", "varShapes":[[1, 2, 2, 5]], "varTypes":["float32"], "varInit":["uniform"], "ksizes":[1,2],"strides":[1,1], "padding":"SAME", "data_format":"NHWC", "output_dtype":tf.int32},
```

Comment:
Need more tests for this.
For example: double, NCHW, int64 datatype, valid padding mode, etc.